.gitignore

```diff
@@ -7,6 +7,10 @@
 
 # testing
 /coverage
+/cypress/videos/*
+!/cypress/videos/.gitkeep
```

Comment:
I'm seeing this as deleted file after running tests 🤔

Reply:
Should be fixed now

- [ ] Fix cypress deleting .gitkeep